mkdocs.yml

```diff
@@ -88,6 +88,7 @@ nav:
 #        - Data: docs/topics/data/README.md
 #        - Configuration: docs/topics/configuration/README.md
         - Workflows: docs/topics/workflows/README.md
+		- Publishing a new release: docs/topics/publishing-releases/README.md
```

Comment:
The number of spaces is important. It is a yaml file.

Reply:
Yeah, it was actually tabs my mistake.